arcgis-ios-sdk-samples/Scenes/Get Elevation at a point/GetElevationPoint.storyboard

```diff
@@ -0,0 +1,87 @@
+                            <constraint firstItem="gI1-ae-TH7" firstAttribute="leading" secondItem="Yv5-8d-hM0" secondAttribute="leading" id="iO8-Dk-ha9"/>
+                            <constraint firstItem="TbC-bb-iKU" firstAttribute="bottom" secondItem="p32-Us-wmg" secondAttribute="bottom" constant="-100" id="inh-fy-AjV"/>
+                            <constraint firstAttribute="bottom" secondItem="gI1-ae-TH7" secondAttribute="bottom" id="kLx-6e-wy0"/>
+                        </constraints>
+                        <viewLayoutGuide key="safeArea" id="p32-Us-wmg"/>
+                    </view>
+                    <connections>
+                        <outlet property="elevationPointLabel" destination="TbC-bb-iKU" id="fJi-sg-fIa"/>
+                        <outlet property="sceneView" destination="gI1-ae-TH7" id="4yL-od-R8q"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="Kc0-vp-knm" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="125" y="-23"/>
+        </scene>
+        <!--Elevation View Controller-->
+        <scene sceneID="mbo-WU-56s">
+            <objects>
+                <viewController storyboardIdentifier="ElevationViewController" automaticallyAdjustsScrollViewInsets="NO" id="0rn-Ar-HUQ" userLabel="Elevation View Controller" customClass="ElevationViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" opaque="NO" contentMode="scaleToFill" id="c9Z-Hn-Ppo" userLabel="Label View">
+                        <rect key="frame" x="0.0" y="0.0" width="310" height="20"/>
+                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                        <subviews>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="jNi-uF-fm7" userLabel="Elevation Label">
+                                <rect key="frame" x="5" y="5" width="310" height="10"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                        </subviews>
+                        <color key="backgroundColor" white="0.66666666666666663" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
```

Comment:
You may have noticed that this doesn't apply to the popover's arrow. Please set the `backgroundColor` property of the popover presentation controller instead.